backend/api.py

```diff
@@ -311,4 +316,83 @@ async def websocket_pcap(websocket: WebSocket):
                 continue
 
     except WebSocketDisconnect:
-        print("Client disconnected from /ws/gnb")
\ No newline at end of file
+        print("Client disconnected from /ws/gnb")
+
+@app.get("/gnb/config")
+def show_gnb_config():
+    """
+    Return a list of all .yaml or .conf files that contain 'gnb' in their name
+    from the predefined configuration directory.
+
+    Returns:
+        dict: A dictionary with a list of matching file names.
+    """
+    try:
+        if not CONFIG_DIR.exists():
+            raise HTTPException(status_code=404, detail="Configuration directory not found.")
+
+        files = [
+            f.name for f in CONFIG_DIR.iterdir()
+            if f.is_file() and "gnb" in f.name.lower() and f.suffix in [".yaml", ".conf"]
+        ]
+
+        return {"files": files}
+
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Failed to list config files: {str(e)}")
+
+@app.post("/metrics/push")
+def push_metrics():
+    """
+    Collects current system CPU and memory usage metrics and writes them to the InfluxDB bucket.
+    This is a test version to demonstrate InfluxDB integration with FastAPI.
+
+    Returns:
+        dict: A confirmation message including the pushed CPU and memory usage percentages.
+    """
+    cpu_percent = psutil.cpu_percent()
+    mem = psutil.virtual_memory()
+    mem_percent = mem.percent
+
+    point = (
+        Point("system_usage")
+        .field("cpu", cpu_percent)
+        .field("memory", mem_percent)
+    )
+
+    write_api.write(bucket=INFLUXDB_BUCKET, org=INFLUXDB_ORG, record=point)
+
+    return {"status": "Metrics pushed", "cpu": cpu_percent, "memory": mem_percent}
+
+@app.websocket("/ws/metrics/latest")
```

Comment:
this should have a parameters that is the metrics named associated to a specific UE